src/foam/nanos/notification/email/DAOEmailService.js

```diff
@@ -123,10 +123,16 @@ if (foam.util.SafetyUtil.isEmpty(emailMessage.getSubject())) {
   emailMessage.setSubject(templateSubject.render(model));
 }
 
+// If emailTemplate has a send To address use that.
+String sendTo = emailTemplate.getSendTo();
+if (sendTo != null && !foam.util.SafetyUtil.isEmpty(sendTo)) {
```

Comment:
Sorry if I wasn't clear on this earlier.  Similar to displayName, if the emailMessage does not have a sendTo, then use the template.